### PR TITLE
🔖(marion) bump to version 0.4.0

### PR DESCRIPTION
**Pull request by jbpenrath**
# Marion

### Added
- Add Django 4.0 compatibility

### Fixed
- Use static storage class to open static files

### CODE REVIEW

**Review by quitterie-lcs — COMMENTED** [review]
Not sure of that but with the fix you made, don't you need to update the `howard` configuration to require at least `django-marion` 0.4.0 ?

**Comment by quitterie-lcs** [comment]
@jbpenrath I also took the liberty of changing the description of this PR to keep the template we use for the releases

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,11 +8,13 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
 
 ## [Unreleased]
 
+## [0.4.0] - 2022-08-04
```
Comment (carofun) [inline CHANGELOG.md:11]:
```suggestion
## [0.4.0] - 2022-08-05
```
our bad

**Review by carofun — APPROVED** [review]

**Comment by jbpenrath** [comment]
> Not sure of that but with the fix you made, don't you need to update the `howard` configuration to require at least `django-marion` 0.4.0 ?

Oh yes, good catch @quitterie-lcs. Is it ok for you make this change within this PR and release a new version of howard at the same time ?

**Comment by quitterie-lcs** [comment]
> > Not sure of that but with the fix you made, don't you need to update the `howard` configuration to require at least `django-marion` 0.4.0 ?
> 
> Oh yes, good catch @quitterie-lcs. I think I have to address this point in a separate PR as currently django-marion has not been released, I think a ci job will fail.

Indeed, I did not think about the CI job :smile: 
In consequence, a new `howard` release will be necessary

**Review by quitterie-lcs — APPROVED** [review]

**Pull request merged**